parents/integrationtest/pom.xml

```diff
@@ -355,7 +524,9 @@
                 </property>
             </activation>
             <properties>
-                <test.elasticsearch.run.skip>true</test.elasticsearch.run.skip>
+                <!-- Skipping to start all containers here -->
+                <!-- When the uris property is set, usually we don't need any container (neither ES nor others). -->
+                <test.containers.run.skip>true</test.containers.run.skip>
```

Comment:
If you follow my suggestion regarding the configuration properties, you can revert this particular change.